docs/source/developer_guides/mixed_models.md

```diff
@@ -28,12 +26,12 @@ peft_model.load_adapter(<path_to_adapter2>, adapter_name="other")
 peft_model.set_adapter(["default", "other"])
 ```
 
-The last line is necessary if you want to activate both adapters, otherwise, only the first adapter would be active. Of course, you can add more different adapters by calling `add_adapter` repeatedly.
+The [`~PeftMixedModel.set_adapter`] method is necessary to activate both adapters, otherwise only the first adapter would be active. You can keep adding more adapter by calling [`~PeftModel.add_adapter`] repeatedly.
```

Comment:
typo line 29: "You can keep adding more **adapter**" -> "You can keep adding more **adapters**"